src/components/ShareTrajectoryButton/index.tsx

```diff
@@ -19,15 +19,19 @@ const ShareTrajectoryButton = ({
     simulariumFile,
 }: ShareTrajectoryButtonProps): JSX.Element => {
     const [isSharing, setIsSharing] = React.useState(false);
+    const [tooltipVisible, setTooltipVisible] = React.useState(false);
 
     const trajectoryIsSharable = isOnlineTrajectory(location.href);
 
     const handleShare = () => {
         setIsSharing(!isSharing);
+        // this is to ensure the tooltip closes when the button is clicked
+        setTooltipVisible(false);
```

Comment:
This doesn't work all the time, if you move the cursor on and click quickly `handleShare` runs during the `mouseOver` delay and then `onOpenChange` opens the tooltip while the modal is open. Suggestion below seems to fix it, probably other ways as well.
```suggestion
         if (isSharing) return;
        setTooltipVisible(false);
```